db/migrate/20240206140353_create_swag_requests.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSwagRequests < ActiveRecord::Migration[7.2]
+  def change
+    create_table :hackathon_swag_requests do |t|
+      t.references :hackathon, null: false, foreign_key: true
+      t.references :mailing_address, null: false, foreign_key: true
+
+      t.timestamp :delivered_at
+
+      t.timestamps
+    end
+
+    Hackathon.where.associated(:swag_mailing_address).includes(:swag_mailing_address).find_each do |hackathon|
+      Hackathon::SwagRequest::Delivered.suppress do
+        hackathon.create_swag_request!(mailing_address: hackathon.swag_mailing_address.attributes)
```

Comment:
Would this create a brand new mailing address? Maybe we re-use the old one

Reply:
[We are.](https://github.com/hackclub/hackathons-backend/pull/426/files#:~:text=belongs_to%20%3Amailing_address)